src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -6096,6 +6096,27 @@ inline void compute_separable_horizontal_resample(Rpp32f *inputPtr, T *outputPtr
     }
 }
 
+inline void compute_jitter_src_loc_sse(__m128i *pxXorwowStateX, __m128i *pxXorwowStateCounter, __m128 &pRow, __m128 &pCol, __m128 &pKernelSize, __m128 &pBound, __m128 &pHeightLimit, __m128 &pWidthLimit, __m128 &pStride, __m128 &pChannel, Rpp32s *srcLoc)
+{
+    __m128 pRngX = rpp_host_rng_xorwow_4_f32_sse(pxXorwowStateX, pxXorwowStateCounter);
```

Comment:
Can we use AVX instead of SSE here?
https://github.com/r-abishek/rpp/blob/f9a7063d19c06cc9e273353fd1d2b5719cedcaae/src/include/cpu/rpp_cpu_common.hpp#L338

Applicable to all code in this function